server/app/routes/files.js

```diff
@@ -0,0 +1,12 @@
+const express = require('express');
+const expressFileUpload = require('express-fileupload');
+
+const { upload } = require('../controllers/files');
+const { protect } = require('../middleware/auth');
+
+const router = express.Router();
+
+router.use(expressFileUpload());
+router.post('/upload', protect, upload);
```

Comment:
Instead of '/upload', perhaps '/' is good enough?
With '/upload' the API looks like this:
``
POST /api/v1/files/upload
``
This says that we're creating an upload, which doesn't sound quite right

I think
``
POST /api/v1/files
``
makes more sense, as it says we're creating a file